platformio.ini

```diff
@@ -87,6 +87,8 @@ build_flags =
   -D AO_DBG_LEVEL=AO_DL_INFO
   -D AO_MG_VERSION_614 ; use MG v6.14
   -D AO_CA_CERT_LOCAL ; manage certificate locally
+  -D ARDUINO_LMIC_PROJECT_CONFIG_H_SUPPRESS
+  -D hal_init=LMICHAL_init ; Workaround mcci arduino-lmic bug 714 on esp32
```

Comment:
Do these apply to all builds? May be better to put them in the LORA specific env, or group all the LORA build env in to a new var that can be included in the envs:

```
build_flags_lora =
  -D ARDUINO_LMIC_PROJECT_CONFIG_H_SUPPRESS
  -D hal_init=LMICHAL_init ; Workaround mcci arduino-lmic bug 714 on esp32
```

then in the env:

```
build_flags =
  ${common.build_flags}
  ${common.src_build_flags}
  ${common.version}.dev
  ${common.build_flags_lora}
  -D DEBUG_PORT=Serial
  -D WIFI_LED=25
```

Reply:
They're generic fixes for the LMIC code, but good point on moving them to lora specific builds.  I can do the same for the other things like the country stuff too